packages/offix-cache/src/createOptimisticResponse.ts

```diff
@@ -49,9 +50,16 @@ export const createOptimisticResponse = (options: OptimisticOptions) => {
     __typename: "Mutation"
   };
 
+  let mappedVariables = variables;
+
+  if (options.inputMapper) {
```

Comment:
perhaps we could do the following?
```suggestion
  if (options.inputMapper && typeof a === 'function') {
```